src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -1000,7 +1000,7 @@ class CreateVmModal extends React.Component {
             // then show errors in the notification area
             this.setState({ createMode: startVm ? RUN : EDIT, validate: false });
 
-            const unattendedInstallation = this.state.rootPassword || this.state.userLogin || this.state.userPassword;
+            const unattendedInstallation = !(this.state.sourceType === CLOUD_IMAGE) && (this.state.rootPassword || this.state.userLogin || this.state.userPassword);
```

Comment:
This is easier to read with `this.state.sourceType !== CLOUD_IMAGE`, as it otherwise isn't clear at first sight whether or not this applies to the whole condition.

Reply:
So you have decided not to change this?